### PR TITLE
feat(conference): completed alpescraft video by adding speakers and time

### PR DESCRIPTION
**Pull request by Boyadjie**
## 🤔 Why do you want to make those changes?

AlpesCraft also want to present there speakers -> #147 

## 🧑‍🔬 How did you make them?

I've added a condition that if there is a speaker data, it will not show the logo but the speaker.s. Same for the time information.

## 🧪 How to check them?

- You can change the data on src/video and see the differences when running `pnpm start`

### CODE REVIEW

**Review by Slashgear — COMMENTED** [review]
🆗 With lint warning fixed 

Maybe we could add this to the website page for conference ?

File: src/Video.tsx [hunk 1]
```diff
@@ -161,13 +162,26 @@ export const RemotionVideo: React.FC = () => {
...
+								{
+									name: 'Antoine Caron',
+									picture:
+										'https://ca.slack-edge.com/T108ZKPMF-U5M2F1A69-36e4b6574de8-192',
+								},
+							],
 							date: '1 et 2 Juin 2023',
+							time: '14h00',
 							location: 'Grenoble - Alpexpo',
 						}}
```
Comment (CruuzAzul) [inline src/Video.tsx:186]: Did you check if you can't use : `defaultTalkValues` ?

File: src/conference/alpescraft/AlpesCraft.tsx [hunk 2]
```diff
@@ -38,10 +48,14 @@ export const AlpesCraft: React.FC<AlpesCraftProps> = ({
...
-				<Logo />
+				{speakers ? <Speakers speakers={speakers} /> : <Logo />}
 			</Sequence>
 			<Sequence name="Details" from={60}>
-				<Details date={date} location={location} />
+				{time ? (
+					<Details date={date} time={time} location={location} />
+				) : (
+					<Details date={date} location={location} />
+				)}
```
Comment (CruuzAzul) [inline src/conference/alpescraft/AlpesCraft.tsx:58]: Maybe one `<Details />` component with the condition in the props declaration ?

Comment (Boyadjie) [reply src/conference/alpescraft/AlpesCraft.tsx:58]: There is but if you don't add the time information it will add an null time on the video

Comment (Slashgear) [reply src/conference/alpescraft/AlpesCraft.tsx:58]: Let's change Type of props of the details component to handle nullable attributes.
The condition should be in Details component not in Alpescraft

**Review by Slashgear — APPROVED** [review]
otherwise 👍🏻

File: src/conference/alpescraft/AlpesCraft.tsx (hunk 2 above)
Comment (Boyadjie) [reply src/conference/alpescraft/AlpesCraft.tsx:58]: okay

**Review by CruuzAzul — APPROVED** [review]

**Pull request merged**